packages/maskbook/src/plugins/Wallet/apis/opensea.ts

```diff
@@ -123,7 +123,6 @@ export async function getAssetsList(from: string, size = 50, page = 0) {
     const response = await fetch(`https://api.opensea.io/api/v1/assets?${params.toString()}`, {
         method: 'GET',
         mode: 'cors',
-        cache: 'force-cache',
```

Comment:
I owned my new first asset. But with `cache: 'force-cache'`, it still loads empty array.